tools/run_tests/run_tests.py

```diff
@@ -712,7 +713,7 @@ def test_specs(self):
         runtime_cmd = ['mono']
 
     specs = []
-    for assembly in tests_by_assembly.iterkeys():
```

Comment:
this one escaped my eyes iter(tests_by_assemble.keys()) should yield the same results in both